### PR TITLE
OSD-19605 a script to extract email from pull secret

### PR DESCRIPTION
**Pull request by feichashao**
### What type of PR is this?

Feature

### What this PR does / Why we need it?
In some cases, SRE need to validate if the email in the pull secret matches the record in OCM.
Instead of pulling down the how secret file, we can use this script to only get the info needed for validation.

### Which Jira/Github issue(s) does this PR fix?
Part of [OSD-19605](https://issues.redhat.com//browse/OSD-19605)

### Special notes for your reviewer
The test result looks like this:
```
ocm backplane testjob logs openshift-job-dev-wnrbj 
siwu@redhat.com
```
It can be run on both classic OSD and Hypershift clusters.

### Pre-checks (if applicable)

- [x] Validated the changes in a cluster
- [ ] Included documentation changes with PR

### CODE REVIEW

File: scripts/SREP/get-pull-secret-email/script.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```
Comment (bmeng) [inline scripts/SREP/get-pull-secret-email/script.sh:1]: The existing osdctl subcommand https://github.com/openshift/osdctl/blob/master/cmd/cluster/validatepullsecret.go#L56 will check the ocm registry credential as well. It seems we cannot do the same in managed scripts?

Comment (feichashao) [reply scripts/SREP/get-pull-secret-email/script.sh:1]: For the OCM part we still need to do that in `osdctl`.

**Comment by bmeng** [comment]
Could you add a simple readme file for this script?

**Comment by feichashao** [comment]
Added a README

**Pull request merged**